metaseq/models/transformer.py

```diff
@@ -797,13 +846,29 @@ def buffered_future_mask(self, tensor):
             if self.use_alibi:
                 alibi = self.alibi.repeat(batch_size, 1, 1)  # batch_size, 1, 1
                 self._future_mask = self._future_mask.unsqueeze(0) + alibi
+            elif self.self_attn_doc_sep != UNSPECIFIED_DOC_SEP:
```

Comment:
Should this conditional be based on not using alibi? Or be standalone?

Reply:
Good point, fixed this.